src/index.ts

```diff
@@ -5,12 +5,14 @@ import debug from 'debug'
 import type { FileSystem, ResolveOptions, Resolver } from 'enhanced-resolve'
 import enhancedResolve from 'enhanced-resolve'
 import { hashObject } from 'eslint-module-utils/hash.js'
-import { sync as globSync } from 'fast-glob'
+import FastGlob from 'fast-glob'
```

Comment:
```suggestion
import fg from 'fast-glob'
```

https://github.com/mrmlnc/fast-glob?tab=readme-ov-file#synchronous

It seems `fg.globSync` is available.